src/components/Backdrop/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import './style.scss';
+import scenery from 'assets/common_backdrop/scenery.svg';
+import sceneryMobile from 'assets/common_backdrop/scenery_mobile.svg';
+import background from 'assets/common_backdrop/background.png';
+import topSign from 'assets/common_backdrop/top-sign.svg';
+import NavBar from 'components/NavBar';
+
+const signRotations = ['9.03deg', '0deg', '-8deg', '0deg'];
```

Comment:
Wow. 9.03deg. Very precise. 9deg is good enough for the work we're doing here.

Reply:
I think it's good practice to put this const inside the function. It's generally a good thing to put variables closer to where you use them